lib/ansiblelint/utils.py

```diff
@@ -225,7 +225,7 @@ def _include_children(basedir, k, v, parent_type):
     return [{'path': result, 'type': parent_type}]
 
 
-def _taskshandlers_children(basedir, k, v, parent_type):
+def _taskshandlers_children(basedir, k, v, parent_type):  # noqa: C901
```

Comment:
This and similar changes don't belong here. Use a separate PR for moving ignores from the flake8 config to files.